modules/preparing-an-inital-cluster-deployment-for-mce-connected.adoc

```diff
@@ -57,7 +57,7 @@ The installer does not validate extra manifests.
     name: multicluster-engine
     namespace: multicluster-engine
   spec:
-    channel: "stable-2.1"
+    channel: "stable-2.3"
```

Comment:
This change is based on [this bundle](https://catalog.redhat.com/software/containers/multicluster-engine/mce-operator-bundle/6160406290fb938ecf6009c6?architecture=amd64&image=6537e7c0fdfcb606c55e382f&container-tabs=dockerfile), where lines 8 and 14 suggest that the `stable-2.3` channel is valid for versions 4.13 and 4.14.

I'll likely need to review this example yaml again for the 4.15 docs.